### DISCUSSION TITLE
美国的vps使用hy2无法使用ChatGPT

### DISCUSSION THREAD

**Discussion by 1017737237** [apernet/hysteria#1213] [Q&A]
同一台美国vps使用其他协议能正常使用ChatGPT，另外有意思的一点香港和台湾vps搭建hy2可以正常使用ChatGPT
![IMG_6309](https://github.com/user-attachments/assets/54191fa2-5e68-4b3f-b28a-33a977426a9a)

**Comment by ghost** [comment]
您好，此問題與代理工具無關，是您的出口節點IP地址被該服務拉黑導致的。
目前ChatGPT對於服務的限制有三種情況，即 網頁版無法使用 應用程式無法使用 兩者都無法使用。
您只需要換一個環境就可以使用了。

**Comment by ghost** [comment]
如果其他協議可以正常使用，那代表您的出口節點同時存在IPv4和IPv6位址，這說明其中之一沒有被拉黑。您只需要讓服務走其中之一的連線即可。

**Reply by 1017737237** [reply]
好的 多谢大佬解答
